test/comparison.test.js

```diff
@@ -0,0 +1,40 @@
+const percySnapshot = require('@percy/webdriverio');
+var minimal_constants = require('../helper/minimal.constants');
+var common_constants = require('../helper/common.constants');
+var comparison_constants = require('../helper/comparison.constants');
+
+console.log('COMPARISON_URL')
+console.log(minimal_constants.COMPARISON_URL); 
+console.log(comparison_constants.COMPARISON_NAME); 
+
+describe('Percy Comparison', () => {
```

Comment:
Same thing here as the baseline test.

[percy] Percy has started!
[percy] Running "wdio run wdio.conf.js --spec test/comparison.test.js"

Execution of 1 workers started at 2022-10-27T00:35:32.538Z

[0-0] COMPARISON_URL
[0-0] https://k8s.bsstag.com
[0-0] Browserstack Staging Comparison to Production
[0-0] RUNNING in firefox - /test/comparison.test.js
[0-0] Error in "Percy Comparison snapshot homepage"
Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
    at <Jasmine>
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[percy] Snapshot taken: Homepage Page
[percy] Snapshot taken: Pricing Page
[percy] Snapshot taken: Automation Integrations Page
[percy] Snapshot taken: Docs Page
[0-0] FAILED in firefox - /test/comparison.test.js

 "spec" Reporter:
------------------------------------------------------------------
[firefox 103.0.2 mac #0-0] Running: firefox (v103.0.2) on mac
[firefox 103.0.2 mac #0-0] Session ID: 5b61b461-522b-401f-b267-d24f5403c935
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] » /test/comparison.test.js
[firefox 103.0.2 mac #0-0] Percy Comparison
[firefox 103.0.2 mac #0-0]    ✖ snapshot homepage
[firefox 103.0.2 mac #0-0]    ✓ snapshot pricing
[firefox 103.0.2 mac #0-0]    ✓ snapshot integrations
[firefox 103.0.2 mac #0-0]    ✓ snapshot docs
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] 3 passing (26.4s)
[firefox 103.0.2 mac #0-0] 1 failing
[firefox 103.0.2 mac #0-0]
[firefox 103.0.2 mac #0-0] 1) Percy Comparison snapshot homepage
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0] Error: Timeout - Async function did not complete within 10000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
[firefox 103.0.2 mac #0-0]     at <Jasmine>
[firefox 103.0.2 mac #0-0]     at listOnTimeout (node:internal/timers:559:17)
[firefox 103.0.2 mac #0-0]     at processTimers (node:internal/timers:502:7)


Spec Files:      0 passed, 1 failed, 1 total (100% completed) in 00:00:31